src/Heading.js

```diff
@@ -0,0 +1,27 @@
+import Text from './Text'
+
+const Heading = Text.withComponent('h3')
+
+Heading.displayName = 'Heading'
+
+Heading.defaultProps = { fontSize: 4, m: 0 }
```

Comment:
nit: I think this is a little more readable broken into multiple lines

Reply:
Yeah, generally multiple lines are cleaner to read & add/remove new object key(s). Updated the code format. Thanks.